src/test/test_rebind.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import subprocess
+import socket
+import os
+import time
+
+def try_connecting_to_socksport():
+    socks_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    if socks_socket.connect_ex(('127.0.0.1', 9052)):
+        tor_process.terminate()
+        print 'FAIL'
+        sys.exit('Cannot connect to SOCKSPort')
+    socks_socket.close()
+
+def wait_for_log(s):
+    while True:
+        l = tor_process.stdout.readline()
+        if s in l:
+            return
+
+if not os.path.exists(sys.argv[1]):
+    sys.exit('ERROR: cannot find tor at %s' % sys.argv[1])
+
+tor_path = sys.argv[1]
+
+tor_process = subprocess.Popen([tor_path,
+                               '-ControlPort', '127.0.0.1:9053', 
```

Comment:
Somewhat risky to use fixed ports here: using fixed ports means that we can't actually run two of these in parallel, which can be a problem in some testing environments (eg running "make check" on two Tor versions at once).